### ISSUE TITLE
Update and face book

### ISSUE THREAD

**Issue by Methen** [Feodor2/Mypal#51]
I have two issues to report one every time there is a update it does not work, and the main one is I can no longer post of facebook
the old version of chrome starting doing the same thing which is the main reason I went to my pal I am running windows xp pro 32 bit please find a way to get this to work properly on face book

**Comment by Methen** [comment]
update the areas I am having problems with in face book is comments and messenger

**Comment by chrischris888** [comment]
That is correct, comments and messenger in Facebook are difficult ... I am using a contr+Copy any character and then contr+Paste+MouseClick simoultaneously ... and after that I can type in comments and messenger but still difficult.

**Comment by chrischris888** [comment]
Is there any chance this month for a MyPal update ? did Feodor resumed his work ? ... the situation is becoming critical on using Facebook and YouTube accounts

**Comment by Feodor2** [comment]
@chrischris888

> Is there any chance this month for a MyPal update ? 

Yes i suppose, at least i have already working quantum on my winxp, yet no support video and direct3d

**Comment by Methen** [comment]
Another thing I would like to see is for this browser to be able to use chrome extensions I have a horrible time with ads on your tube and there is a volume booster that I would love to have on my pal that I use on chrome although sometimes it does not always work

**Comment by chrischris888** [comment]
Thank you very much Feodor for doing such a huge task helping all of us to keep using Windows XP on our computers. I'll wait for the february new update ...

**Comment by chrischris888** [comment]
Since just minutes ago, I cannot access my YouTube account, using MyPal on Windows XP, ... the need for a new MyPal upgrade version is now extremely urgent.

**Comment by NEO-SPECTRUMAN** [comment]
try basilisk/serpent by roytam1

**Comment by NEO-SPECTRUMAN** [comment]
> Yes i suppose, at least i have already working quantum on my winxp, yet no support video and direct3d

what about memory usage?

**Comment by chrischris888** [comment]
I tried latest basilisk/serpent by roytam1 and it does not work on my YouTube account ... the YouTube account asked each time to confirm identity, and after that still don't have access. It does not work, Pale Moon also does not work either. Facebook is working exactly as it still does under the latest updated MyPal (2021 version actually) and without any new updates it will soon not work at all. I hope Feodor2 will solve the problem, as he said that this month it is possible to see a new update which will fix all these issues... I am very worried for the moment.

**Comment by NoNameNeeded2000** [comment]
If he's really capable to make Firefox quantum work nicely on XP, it won't be a simple update for the old version of Mypal but it will be an almost completely new browser, since the ancient version of Firefox that Mypal was based doesn't really have much to do with the more recent versions of Firefox. 
So be prepared for a new Firefox profile. You probably won't be able to use your old Mypal profile with a much more recent version of Firefox.

Nevertheless, I'm really surprised that Quantum even works on XP and I'm looking forward to trying what Feodor will come up with.

**Comment by NEO-SPECTRUMAN** [comment]
>I tried latest basilisk/serpent by roytam1 

55 or 52?

**Comment by Methen** [comment]
Hey I was just wondering has anyone had and problems with missing booknarks on my pal

**Comment by NEO-SPECTRUMAN** [comment]
missing bokmarks - nope
wrong tabs order after session restore - every day
